### PR TITLE
优化主界面布局，删除无用布局文件，添加必要的go文件

### PR DESCRIPTION
**Pull request by floatingrain**
为什么说是“奇怪错误”：
代码刚拉下来的时候，kt文件甚至爆红，libv2ray.aar文件中甚至没有对应函数存在。该项目应该好好考虑下自己的开源状态了，代码pull下来模块之间牛头不对马嘴，go.mod也不存在，类似一系列阻挡构建的操作严重降低他人的参与积极性。目前所有对AndroidLibV2ray的改动似乎都没有正式发布到当前代码版本中，因此我重新构建了libv2ray.aar，主分支如不接受可在后来的review中进行修改。

### CODE REVIEW

**Comment by rurirei** [comment]
1. make sure a clean commit tree as there are no commit like "Merge branch"
2. do not upload your library as do not override official build
3. this is not a open source project with fully and completely

**Review by rurirei — CHANGES_REQUESTED** [review]
delete this commit as is outdated and do not provide your own build.

File: AndroidLibV2rayLite/CoreI/Status.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ type Status struct {
 }
 
 func CheckVersion() int {
-	return 21
+	return 22
```
Comment (rurirei) [inline AndroidLibV2rayLite/CoreI/Status.go]: no outdated commit like this please ..

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,7 @@ V2rayNG/app/src/main/java/com/v2ray/ang/InappBuyActivity.java
 *.jks
 V2rayNG/app/release/output.json
 .idea/
-.gradle/
\ No newline at end of file
+.gradle/
+V2rayNG/app/release/
+
+V2rayNG/app/libs/
```
Comment (rurirei) [inline .gitignore]: do not add this as we need to release official build.

File: .gitignore [hunk 3]
```diff
@@ -6,4 +6,7 @@ V2rayNG/app/src/main/java/com/v2ray/ang/InappBuyActivity.java
 *.jks
 V2rayNG/app/release/output.json
 .idea/
-.gradle/
\ No newline at end of file
+.gradle/
+V2rayNG/app/release/
```
Comment (rurirei) [inline .gitignore]: delete this as using gradle to build app please, not this bad path from AS.

File: V2rayNG/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,124 +1,101 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (rurirei) [inline V2rayNG/app/src/main/res/layout/activity_main.xml:2]: review requested @yuhan6665

**Review by rurirei — CHANGES_REQUESTED** [review]
how deep work you are though, please clean up your pull request completely before do next commit.

**Comment by floatingrain** [comment]
> 1. make sure a clean commit tree as there are no commit like "Merge branch"
> 2. do not upload your library as do not override official build
> 3. this is not a open source project with fully and completely

1. I have no interest in this OCD but not matterd thing.
2. will fix.
3. absolutely不可理解。

File: AndroidLibV2rayLite/CoreI/Status.go (hunk 1 above)
Comment (floatingrain) [reply AndroidLibV2rayLite/CoreI/Status.go]: > no outdated commit like this please ..

这是什么？原仓库代码就是这样啊，我没有在此处修改。

**Comment by rurirei** [comment]
> > 1. make sure a clean commit tree as there are no commit like "Merge branch"
> > 2. do not upload your library as do not override official build
> > 3. this is not a open source project with fully and completely
> 
> 1. I have no interest in this OCD but not matterd thing.
> 2. will fix.
> 3. absolutely不可理解。

we won't merge uncleared commits.

**Comment by rurirei** [comment]
> > 1. make sure a clean commit tree as there are no commit like "Merge branch"
> > 2. do not upload your library as do not override official build
> > 3. this is not a open source project with fully and completely
> 
> 1. I have no interest in this OCD but not matterd thing.
> 2. will fix.
> 3. absolutely不可理解。

1. we have no interest to merge your uncleaned and uncleared commits
3. #651

**Comment by floatingrain** [comment]
> > > 1. make sure a clean commit tree as there are no commit like "Merge branch"
> > > 2. do not upload your library as do not override official build
> > > 3. this is not a open source project with fully and completely
> > 
> > 1. I have no interest in this OCD but not matterd thing.
> > 2. will fix.
> > 3. absolutely不可理解。
> 
> 1. we have no interest to merge your uncleaned and uncleared commits
> 3. #651 

所有必要更正已完成，如果有权限，主界面更改可迅速合并，调整了原来大量的混乱内容，并且增大字体，增加了log的可读性。 toolbar重新与下方测试栏进行宽度匹配，以产生对称视觉效果。

File: AndroidLibV2rayLite/Process/Escort/escort.go [hunk 5]
```diff
@@ -1,11 +1,10 @@
 package Escort
 
 import (
+	"log"
 	"os"
 	"os/exec"
 
-	"log"
```
Comment (rurirei) [inline AndroidLibV2rayLite/Process/Escort/escort.go:7]: could you try to clean this outdated commit, did you really based on latest upstream?

File: AndroidLibV2rayLite/go.mod [hunk 6]
```diff
@@ -0,0 +1,11 @@
+module github.com/2dust/AndroidLibV2rayLite
+
+go 1.14
```
Comment (rurirei) [inline AndroidLibV2rayLite/go.mod:3]: golang 1.15+ required

File: V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayServiceManager.kt [hunk 7]
```diff
@@ -310,11 +310,12 @@ object V2RayServiceManager {
...
                         var proxyTotal = 0L
                         val text = StringBuilder()
-                        outboundTags.forEach {
+                        outboundTags.forEach<String> {
                             val up = v2rayPoint.queryStats(it, "uplink")
                             val down = v2rayPoint.queryStats(it, "downlink")
                             if (up + down > 0) {
-                                appendSpeedString(text, it, up / sinceLastQueryInSeconds, down / sinceLastQueryInSeconds)
+                                appendSpeedString(text, it, up / sinceLastQueryInSeconds, down /
+                                        sinceLastQueryInSeconds)
```
Comment (rurirei) [inline V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayServiceManager.kt:318]: unnecessary line wrap

File: V2rayNG/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -1,124 +1,101 @@
...
-
-            <android.support.v7.widget.Toolbar
-                android:id="@+id/toolbar"
-                android:layout_width="match_parent"
-                android:layout_height="?attr/actionBarSize"
-                android:background="?attr/colorPrimary"
-                app:popupTheme="@style/AppTheme.PopupOverlay" />
-
-        </android.support.design.widget.AppBarLayout>
+        android:fitsSystemWindows="true">
```
Comment (rurirei) [inline V2rayNG/app/src/main/res/layout/activity_main.xml:8]: `false` as v2rayNG do not need to overide statusBar

File: V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayServiceManager.kt (hunk 7 above)
Comment (rurirei) [inline V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayServiceManager.kt]: unnecessary line wrap

File: AndroidLibV2rayLite/go.mod (hunk 6 above)
Comment (rurirei) [inline AndroidLibV2rayLite/go.mod]: golang 1.15+ required

**Review by rurirei — APPROVED** [review]
how much hard work you made!

**Comment by floatingrain** [comment]
> how much hard work you made!

ALL DONE

**Review by rurirei — APPROVED** [review]
thanks for your hard work. review requested for big changes in UI @yuhan6665

**Comment by floatingrain** [comment]
你认为怎么样 @yuhan6665 ，我让它看起来至少没那么粗糙了

**Comment by rurirei** [comment]
...

1. you can make pull request at other branch as you can do your work at your own master branch
2. make your commit tree clean as possible at your next pull request

**Pull request closed without merging**